src/components/VSelect/mixins/select-autocomplete.js

```diff
@@ -30,32 +34,41 @@ export default {
         i, this.searchValue, this.getText(i))
       )
     },
-    onKeyDown (e) {
-      // If enter or space is pressed, open menu
-      if (!this.menuIsActive &&
-        [13, 32, 38, 40].includes(e.keyCode)
-      ) {
+    async onKeyDown (e) {
```

Comment:
no component level async calls please, something we can explore in the future

Reply:
Is this because of a-la-carte, or do you just not like having random promises all over the place?